src/test/suite/fixtures/recorded/surroundingPair/takeCoreRiskPastJustHarp.yml

```diff
@@ -0,0 +1,45 @@
+languageId: plaintext
```

Comment:
This test actually worked before, but seemed worth having a test for the code that downgrades to weak target